web/templates/note/form.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= form_for @changeset, @action, fn f -> %>
+<div>
+  <%= text_input f, :title , placeholder: "Title" %>
```

Comment:
Avoid using placeholders as labels. This is a good read on the subject: https://www.nngroup.com/articles/form-design-placeholders/